src/common/api/misc.ts

```diff
@@ -71,3 +71,19 @@ export const fetchGif = async (query: string | null, limit: string, offset: stri
   }
   return gifs;
 };
+
+export const marketInfo = async (): Promise<number> => {
+
+  const url = `https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=hive%2C%20hive_dollar&order=market_cap_desc&per_page=100&page=1&sparkline=false`;
+  const data = await axios.get(url)
+  .then((r: any) => r.data)  
+  return data;
+};
+
+export const marketChart = async (token: string): Promise<number> => {
+
+  const url = `https://api.coingecko.com/api/v3/coins/${token}/market_chart?vs_currency=usd&days=30`;
+  const data = await axios.get(url)
+  .then((r: any) => r.data)
+  return data;
```

Comment:
Please, do not use `async/await` and `then` constructions together. 
It should be:
```
const { data } = await axis.get(url);
return data;
```